samples/algorithms/sudoku-grover/ColoringGroverWithConstraints.qs

```diff
@@ -30,8 +30,8 @@ namespace Microsoft.Quantum.Samples.ColoringGroverWithConstraints {
     /// ## register
     /// The register of qubits to be measured.
     operation MeasureColoring (bitsPerColor : Int, register : Qubit[]) : Int[] {
-        let numVertices = Length(register) / bitsPerColor;
-        let colorPartitions = Partitioned(ConstantArray(numVertices - 1, bitsPerColor), register);
+        let nVertices = Length(register) / bitsPerColor;
+        let colorPartitions = Partitioned(ConstantArray(nVertices - 1, bitsPerColor), register);
```

Comment:
Minor, but since this was written, the new `[foo, size=n]` syntax was introduced. While updating, it may be helpful to use that notation here in place of `ConstantArray`:

```suggestion
        let colorPartitions = Partitioned([bitsPerColor, size=nVertices - 1], register);
```